### ISSUE TITLE
Cannot compile Vault app

### ISSUE THREAD

**Issue by hailq** [aragon/aragon-apps#393]
I tried to compile app and got this error: 
```
@aragon/apps-vault/contracts/Vault.sol:31:30: TypeError: Trying to create an instance of an abstract contract.
        erc20ConnectorBase = new ERC20Connector();
```

```
@aragon/apps-vault/contracts/Vault.sol:32:28: TypeError: Trying to create an instance of an abstract contract.
        ethConnectorBase = new ETHConnector();
```

**Comment by onetom** [comment]
Also reported via https://github.com/aragon/aragen/issues/21

**Comment by sohkai** [comment]
This will be fixed with #292.
